extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php

```diff
@@ -4,6 +4,17 @@ class PortableInfoboxBuilderHooks {
 	const INFOBOX_BUILDER_SPECIAL_PAGE = 'Special:InfoboxBuilder';
 
 	/**
+	 * Adds infobox builder helper js assets to Template Classification on Edit page
+	 *
+	 * @return true
+	 */
+	public function onTCAfterEditPageAssets() {
+		\Wikia::addAssetsToOutput( 'portable_infobox_builder_template_classification_helper_js' );
+
+		return true;
```

Comment:
Let's either remove this line or check the value returned here in `TemplateClassification.hooks::onBeforePageDisplay`. I suggest the former one because we don't use it anyway.